src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/probation/model/ProbationSearchResult.kt

```diff
@@ -0,0 +1,15 @@
+package uk.gov.justice.digital.hmpps.createandvaryalicenceapi.service.probation.model
+
+import io.swagger.v3.oas.annotations.media.Schema
+
+@Schema(description = "Describes a probation search result")
```

Comment:
is this returned via our API? if so it should live in `uk/gov/justice/digital/hmpps/createandvaryalicenceapi/model`